### DISCUSSION TITLE
new my-first-branch status for quickstart

### DISCUSSION THREAD

**Discussion by higgn** [community/community#127832] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/171432680/e195dcc7-583a-451d-ad1f-731922affac0)
why not next step what to do

**Comment by ShonHarsh** [comment]
@Zesina,
It appears you need to check in some code on your **my-first-branch** then you will be able to push those changes to GitHub, create a pull request, and merge into main.  The [GitHub Documentation](https://docs.github.com/en/get-started/using-github/github-flow) has more information.

**Reply by higgn** [reply]
Thanks for your kind reply :) I am learning new things ... while some time get worried that Ai will replace me as its op

**Reply by ShonHarsh** [reply]
You are welcome @Zesina !
Source control and GitHub can be confusing at first but it is important so keep practicing.  You will pick it up quickly if you are... committed to it!

**Comment by davevad93** [comment]
Hi @Zesina ,

Your secondary branch is up to date with your `main branch` that means you haven't made any changes. Secondary branches are used for developing new features or fixing bugs without affecting the `main branch` of your repo. For instance, imagine you have pushed the code of your website on your `main branch`, but then you want to add a new feature, so the right thing to do instead of pushing the new changes directly on the `main branch` would be to create a `secondary branch` and push the new changes there, if everything is fine then you can open a `pull request`, solve possible conflicts and finally you can `merge` it into your `main branch` meaning that all the new changes you've made will be integrated into your main code. Resuming, push some code into `my-first-branch` and then open a `pull request` and merge the changes into your `main branch`. For more information about branches take look at this from the docs:

- [About branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches)

**Reply by higgn** [reply]
Thanks sir , Thanks for your kind reply .... I want to be a reliable and good coder

**Reply by davevad93** [reply]
You're welcome man @Zesina , if you have any questions or doubts don't hesitate to ask!
